### ISSUE TITLE
Download manager could be improved

### ISSUE THREAD

**Issue by venkat2305** [LagradOst/CloudStream-3#889]
When I download the same thing with idm on pc , it downloads with max internet speed

With cloud stream app, download speeds doesn't even reach 1MBPS. Hope you can resolve this.

**Comment by RohitKing2001** [comment]
yeah! I have same issue. it's not even half of speed what 1dm gives
